control/planktoscopehat/planktoscope/camera/mqtt.py

```diff
@@ -107,8 +104,19 @@ def run(self) -> None:
                 if (message := mqtt.msg) is None:
                     continue
                 self._receive_message(message)
-                if (status_update := mqtt.read_message()) is not None:
-                    mqtt.client.publish("status/imager", status_update)
+                if message["topic"] == "camera/info" and message["payload"].get("action") == "get":
+                    camera_name = (
+                        self._camera.camera_name
+                        if self._camera is not None
+                        else "Not recognized"
```

Comment:
https://github.com/PlanktoScope/device-backend/blob/44df4f9e285c31d3fd67cd871ba95ad1efa3311f/control/planktoscopehat/planktoscope/camera/hardware.py#L381 already sets `self._camera.camera_name` to `Not recognized` when needed. If you delete the `if self._camera is not None else "Not recognized"`, does the Python linter complain? If not, let's delete these two lines.